example/ios/Example.xcodeproj/project.pbxproj

```diff
@@ -869,7 +875,7 @@
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
 				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
-				CLANG_CXX_LANGUAGE_STANDARD = "gnu++0x";
+				CLANG_CXX_LANGUAGE_STANDARD = "c++17";
```

Comment:
Also these `CLANG_CXX_LANGUAGE_STANDARD` and `EXCLUDED_ARCHS` changes are not visible for me.